scripts/checker.sh

```diff
@@ -3,5 +3,51 @@ set -e
 set -u
 set -o pipefail
 
-# Do the lifeness check exit status 0 -> success | 1 -> fail
-searchd --status 1> /dev/null && exit 0 || exit 1
\ No newline at end of file
+# mountpoint with readiness probe file
+MOUNT=$(realpath "${PROBE_MOUNTPOINT:-/tmp}")
+READY_FILE="${MOUNT}/checker_ready.txt"
+BOOTSTRAP_FILE="${MOUNT}/bootstrapped.txt"
+PRECACHING_SECONDS=900
+
+# get file age in seconds
+function fileAge
+{
+    local fileMod
+    if fileMod=$(stat -c %Y -- "$1")
+    then
+        echo $(( $(date +%s) - fileMod ))
+    else
+        return $?
+    fi
+}
+
+clean_probe_files() {
+        rm -f "${READY_FILE}" || :
+        rm -f "${BOOTSTRAP_FILE}" || :
+}
+
+# source this stuff until here
+[ "$0" = "${BASH_SOURCE[*]}" ] || return 0
+
+# step 1
+# skip probes during the bootstrap of the container, initial rsync
+[ -e "${BOOTSTRAP_FILE}" ] || exit 0
```

Comment:
Do I understand this correctly, we exit with 0 when `${BOOTSTRAP_FILE}` does not exist ?